.gitignore

```diff
@@ -46,3 +46,5 @@ nosetests.xml
 /examples/boards/ppro/xilinx
 /examples/build/xilinx
 
+/SDRAM_Controller
```

Comment:
Sorry, that slipped in when I was working on the SDRAM controller...
It's removed now